source/_addons/mosquitto.markdown

```diff
@@ -55,6 +55,22 @@ This add-on is attached to the Home Assistant user system, so mqtt clients can m
 
 To use the Mosquitto as [broker](/docs/mqtt/broker/#run-your-own), go to the integration page and install the configuration with one click. If you have old MQTT settings available, remove this old integration and restart Home Assistant to see the new one.
 
+
+  #### Using Mosquitto with Hass.io
+  Install the [Mosquitto add-on](https://www.home-assistant.io/addons/mosquitto/) with the default configuration via 'Hass.io > ADD-ON STORE'. (Don't forget to start the add-on & verify that 'Start on boot' is enabled.)
+   Next, create a new user for MQTT via the `Configuration > Users (manage users)`. Restart your HA. (Note: This name cannot be "homeassistant" or "addon")
+   Once back on-line, return to `Configuration > Integrations` and select configure next to `MQTT`.
+   ##### Example of MQTT Integration
```

Comment:
Perhaps put this as `# Example of MQTT Integration` under line 64

Reply:
Resolved.